specs/ScreenCaptureRequested

```diff
@@ -0,0 +1,394 @@
+# Background
```

Comment:
Why are there two documents? One is ScreenCaptureRequested and the other is ScreenCaptureRequested.md

Reply:
Accidently created the first one without the markdown extension. Removing the old one. Thanks for catching that!